src/log-client.js

```diff
@@ -25,6 +25,10 @@ export function createMelindaApiLogClient({melindaApiUrl, melindaApiUsername, me
     return doRequest({method: 'delete', path: `logs/${correlationId}`, params});
   }
 
+  function getLogs(params) {
```

Comment:
getLogsList or getLogsMetadataList would be better function name to this since it does not fetch just logs but list of metadata about available logs

Reply:
PR updated: function name changed to getLogsList.